docker/gcp_pudl_etl.sh

```diff
@@ -48,16 +48,12 @@ function run_pudl_etl() {
 }
 
 function shutdown_vm() {
-    # Copy the outputs to the GCS bucket
-
     upload_file_to_slack $LOGFILE "pudl_etl logs for $ACTION_SHA-$GITHUB_REF:"
-
+    # Shut down the vm instance when the etl is done.
     echo "Shutting down VM."
-    # # Shut down the vm instance when the etl is done.
-    ACCESS_TOKEN=`curl \
+    ACCESS_TOKEN=$(curl \
```

Comment:
I started using `ShellCheck` a bash linter, and apparently backticks are deprecated.